cyral/provider.go

```diff
@@ -171,6 +171,7 @@ func Provider() *schema.Provider {
 			"cyral_role_sso_groups":             resourceRoleSSOGroups(),
 			"cyral_sidecar":                     resourceSidecar(),
 			"cyral_sidecar_credentials":         resourceSidecarCredentials(),
+			"cyral_repository_access_rules":     resourceRepositoryAccessRules(),
```

Comment:
Maybe its a good idea to have this defined near the group of the other `cyral_repository*` resources

Reply:
We should also rename the file name to `resource_cyral_repository_access_rules.go` to match the resource name

Reply:
Done both.